src/App.module.css

```diff
@@ -36,4 +38,12 @@
   border-radius: 50%;
   display: inline-block;
   border: 1px solid gray;
+}
+
+.footer {
+  padding: 8px;
+}
+
+.footer, .footer a {
+  color: grey;
```

Comment:
suggestion(non-blocking): Since we use the same color in multiple places, we could extract it into a CSS variable 🤔